### PR TITLE
Xero Freshdesk

### PR DESCRIPTION
**Pull request by brandonz2892**

### CODE REVIEW

File: connectors/xero.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+4. Experts
+
+5. Recommended recipes
+
+6. Triggers and actions
+
+To see the supported triggers and actions, scroll to the bottom.
+
+App specific behavior
```
Comment (nandithar) [inline connectors/xero.md]: There's no information for this

File: connectors/xero.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
+# Xero
+
```
Comment (nandithar) [inline connectors/xero.md]: We should add an intro for all connector docs. We should talk about Xero and have a link leading to https://www.xero.com/us/

File: connectors/xero.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+## Supported triggers and actions
```
Comment (nandithar) [inline connectors/xero.md]: Can we make a main heading "Working with Xero" and have the remaining sections underneath it?

File: connectors/xero.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+*List of tax types*
+
+
+## Xero API usage limit
+
+According to the [Xero Developer Help Center](https://community.xero.com/developer/question/17181), there is a daily limit of 1000 API calls that a provider can make against a particular Xero organization in a rolling 24 hour period. If you exceed this rate limit you will receive a HTTP 401 response with the message “oauth_problem=rate%20limit%20exceeded&oauth_problem_advice=please%20wait%20before%20retrying%20the%20xero%20api” in the http response body."
+
+
+## Xero - How to find different IDs in Xero
```
Comment (nandithar) [inline connectors/xero.md]: Rename this "How to find different IDs in Xero

File: connectors/xero.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+### What is an ID?
```
Comment (nandithar) [inline connectors/xero.md]: Can we rename this "Xero IDs"

File: connectors/xero.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+An ID is a unique identifier for an object in the API. In Xero, some IDs such as **Contact ID** are required in creating an action and they can be easily obtained using **Search** or other actions. 
+
+### Contact ID
```
Comment (nandithar) [inline connectors/xero.md]: All types of IDs should have "####" instead of "###"

File: connectors/xero.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+# Xero
+[Xero](https://www.xero.com/us/) is a cloud-based accounting software aimed at small to medium-sized businesses. Xero can be used to manage processes like invoicing and bookkeeping.
```
Comment (huilinn) [inline connectors/xero.md]: "...accounting software for small to medium businesses." sounds better

File: connectors/xero.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+Connecting Xero to Workato allows users to sync data between Xero and other applications, as well as automating processes within Xero.
```
Comment (huilinn) [inline connectors/xero.md]: "Workato allows you to synchronize data between Xero and the other apps you use in your organization. For example, you can sync customer data with your CRM app, sales data with your point-of-sales app, or inventory data with your ERP app."

File: connectors/xero.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+## Supported triggers and actions
+
+The latest triggers and actions can be found on the Workato Xero page.
```
Comment (huilinn) [inline connectors/xero.md]: We should add a link to the page.

File: connectors/xero.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+The app page has the following format:
```
Comment (huilinn) [inline connectors/xero.md]: I would take out lines 10-26. This doesn't seem to be adding much value to the Xero connector page.

File: connectors/xero.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+5. Recommended recipes
+
+6. Triggers and actions
+
+To see the supported triggers and actions, scroll to the bottom.
+
+
+## How do I sync products to different Xero accounts? 
+
+Sometimes, when using Xero integration apps, such as the Infusionsoft-Xero integration app, products from the source app are synced to just one Xero account.
```
Comment (huilinn) [inline connectors/xero.md]: Is there a link we can provide to:
- Xero integration apps?
- Infusionsoft-Xero integration app?

@brandonz2892 maybe ask Vera, she would know.

File: connectors/xero.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+
+This typically happens when the respective products do not already exist in Xero, and thus do not exist under the preferred accounts in Xero.
+
+To get products to sync to specific accounts, simply create the products in Xero first, and place them under your preferred accounts. In order for the sync to work properly, the name of the product in Xero should match that in your source app. Workato's integration apps will automatically sync them over to the right accounts.
+
+
+## Sales tax type when using Create/update items action in Xero
```
Comment (huilinn) [inline connectors/xero.md]: Let's take out the capital C: "create/update items action"

File: connectors/xero.md [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+This typically happens when the respective products do not already exist in Xero, and thus do not exist under the preferred accounts in Xero.
+
+To get products to sync to specific accounts, simply create the products in Xero first, and place them under your preferred accounts. In order for the sync to work properly, the name of the product in Xero should match that in your source app. Workato's integration apps will automatically sync them over to the right accounts.
+
+
+## Sales tax type when using Create/update items action in Xero 
+ 
+To find out what types of sales tax to key in when using the **Create/update items** action in Xero, users can click on the URL below the box (highlighted in image). 
+
+![Sales tax list](/assets/images/connectors/xeno/sales-tax-list.png)
```
Comment (huilinn) [inline connectors/xero.md]: Can we get a more recent screenshot?

File: connectors/xero.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
+# Xero
+[Xero](https://www.xero.com/us/) is a cloud-based accounting software aimed at small to medium-sized businesses. Xero can be used to manage processes like invoicing and bookkeeping.
+
+Connecting Xero to Workato allows users to sync data between Xero and other applications, as well as automating processes within Xero.
+
+## Supported triggers and actions
+
```
Comment (huilinn) [inline connectors/xero.md]: We don't usually have spaces between headers and text.

File: connectors/xero.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+5. Recommended recipes
+
+6. Triggers and actions
+
+To see the supported triggers and actions, scroll to the bottom.
+
+
+## How do I sync products to different Xero accounts? 
+
```
Comment (huilinn) [inline connectors/xero.md]: We don't usually have spaces between headers and text.

File: connectors/xero.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+Sometimes, when using Xero integration apps, such as the Infusionsoft-Xero integration app, products from the source app are synced to just one Xero account.
+
+This typically happens when the respective products do not already exist in Xero, and thus do not exist under the preferred accounts in Xero.
+
+To get products to sync to specific accounts, simply create the products in Xero first, and place them under your preferred accounts. In order for the sync to work properly, the name of the product in Xero should match that in your source app. Workato's integration apps will automatically sync them over to the right accounts.
+
+
+## Sales tax type when using Create/update items action in Xero 
+
```
Comment (huilinn) [inline connectors/xero.md]: We don't usually have spaces between headers and text.

File: connectors/xero.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+## Xero API usage limit
+
+According to the [Xero Developer Help Center](https://community.xero.com/developer/question/17181), there is a daily limit of 1000 API calls that a provider can make against a particular Xero organization in a rolling 24 hour period. If you exceed this rate limit you will receive a HTTP 401 response with the message “oauth_problem=rate%20limit%20exceeded&oauth_problem_advice=please%20wait%20before%20retrying%20the%20xero%20api” in the http response body."
+
+
+## How to find different IDs in Xero
+
+### Xero IDs
+An ID is a unique identifier for an object in the API. In Xero, some IDs such as **Contact ID** are required in creating an action and they can be easily obtained using **Search** or other actions. 
+
```
Comment (huilinn) [inline connectors/xero.md]: Can we talk about how to obtain IDs in Xero in general, then provide just one or two examples of how to obtain IDs in Xero? The way to obtain IDs are all the same, really.

File: connectors/xero.md [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+#### Contact ID
+
+**Contact ID** is an unique identifier for each contact in the Xero. It can be found at the end of the URL of a particular contact page. 
+
+![Contact ID](/assets/images/connectors/xeno/contact-id.png)
```
Comment (huilinn) [inline connectors/xero.md]: Can we get a better screenshot?

File: connectors/xero.md (hunk 9 above)
Comment (huilinn) [inline connectors/xero.md]: "..be found on the Xero connector page."

File: connectors/xero.md [hunk 19]
```diff
@@ -0,0 +1,113 @@
...
+
+4. Experts
+
+5. Recommended recipes
+
+6. Triggers and actions
+
+To see the supported triggers and actions, scroll to the bottom.
+
+
```
Comment (huilinn) [inline connectors/xero.md]: This should be under a section called "Xero integration apps"

File: connectors/xero.md [hunk 20]
```diff
@@ -0,0 +1,113 @@
...
+## How do I sync products to different Xero accounts?
```
Comment (huilinn) [inline connectors/xero.md]: Let's take out this one section; it should be under Xero IA troubleshooting in Freshdesk

File: connectors/xero.md [hunk 21]
```diff
@@ -0,0 +1,113 @@
...
+This will lead users to the **Tax type** page (image below) and they will have to key it in manually.
+
+![Tax types list](/assets/images/connectors/xeno/tax-type-list.png)
+
+*List of tax types*
+
+
+## Xero API usage limit
+
+According to the [Xero Developer Help Center](https://community.xero.com/developer/question/17181), there is a daily limit of 1000 API calls that a provider can make against a particular Xero organization in a rolling 24 hour period. If you exceed this rate limit you will receive a HTTP 401 response with the message “oauth_problem=rate%20limit%20exceeded&oauth_problem_advice=please%20wait%20before%20retrying%20the%20xero%20api” in the http response body."
```
Comment (huilinn) [inline connectors/xero.md]: Can we put the error under a formatted newline like this:

```
“oauth_problem=rate%20limit%20exceeded&oauth_problem_advice=please%20wait%20before%20retrying%20the%20xero%20api”
```

Comment (huilinn) [inline connectors/xero.md]: And take out the extra " at the end

**Pull request merged**